WPMedia/fusion-news-theme-blocks
# Tbrands 97 featured image

## Description

Add product featured image 

## Jira Ticket

- [TBRANDS-97](https://arcpublishing.atlassian.net/browse/TBRANDS-97)

## Acceptance Criteria

block migration ac
- [x] add test coverage for image mock data
- [x] Write readme to include purpose
- [x] Update storybook with examples 
- [x] Take in auth token. Set the variable that is getting this environment to be RESIZER_APP_VERSION to be 2 for this feature pack and this mock data
- [x] Set the alt text in the mock image and pull that down. If no alt, then use the fallback in the image component

acceptance criteria ticket 
- [x] Product Featured Image - Arc Block is available to be added to a page in Page Builder Editor.
- [x] The Product Featured Image is available for display from the Arc Commerce Content Source (TBRANDS-94)
- [x] This block will only output the Product Featured Image only.
- [x] Product Featured Image - Arc Block display and styling match designs.
- [x] Product Featured Image - Arc Block is responsive and works across all device types.

## Test Steps

1. Checkout this branch `git checkout TBRANDS-97-featured-image`
2. See feature pack pr https://github.com/WPMedia/arc-themes-feature-pack/pull/311
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/product-featured-image-block`
4. Create a layout of the right rail on commerce 
5. Add the featured image block 

## Effect Of Changes

### Before

- block not available 

### After

with right rail commerce 

desktop: 
<img width="1361" alt="Screen Shot 2022-07-28 at 13 20 40" src="https://user-images.githubusercontent.com/5950956/181610187-5f459fff-94b7-4f3c-b7f1-1fb55e0f738b.png">


mobile 
<img width="745" alt="Screen Shot 2022-07-28 at 13 20 59" src="https://user-images.githubusercontent.com/5950956/181610163-0822f9be-6b1c-4ea2-8482-1e9e2c861e6d.png">



## Dependencies or Side Effects

- Goes with https://github.com/WPMedia/arc-themes-feature-pack/pull/311
-  manifest https://github.com/WPMedia/arc-themes-manifests/pull/118

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
